components/dashboard/src/projects/NewProject.tsx

```diff
@@ -493,13 +493,12 @@ export default function NewProject() {
                     <div>
                         <div className="text-gray-500 text-center w-96 mx-8">
                             Repository not found?{" "}
-                            <a
-                                href="javascript:void(0)"
```

Comment:
**suggestion(non-blocking)**: We have a few more actions in **/plans** masked as links that could benefit from this. However, I wouldn't block this from merging it as we're going to update and replace plans entirly soon.

https://github.com/gitpod-io/gitpod/blob/08954965c673d38e5aa042de9146bf5406929dd8/components/dashboard/src/settings/Plans.tsx#L390-L396

https://github.com/gitpod-io/gitpod/blob/08954965c673d38e5aa042de9146bf5406929dd8/components/dashboard/src/settings/Plans.tsx#L452-L458

...